src/FakeItEasy.Analyzer/FakeItEasy.Analyzer.nuspec

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>FakeItEasy.Analyzer</id>
+    <version>0.0.0</version>
+    <title>FakeItEasy.Analyzer</title>
+    <authors>Thomas Levesque, FakeItEasy contributors</authors>
+    <description>Provides diagnostic analyzers to warn about incorrect usage of FakeItEasy.</description>

```

Comment:
Perhaps we should add something like "Works in VS 2015 Update 1 and later versions."?